.github/workflows/deploy_rovecomm.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Create RoveComm Release
+
+on:
+  push:
+    tags:
+      - "v*"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: read
+  packages: read
+  id-token: write
+
+concurrency:
+  group: "create-release"
+  cancel-in-progress: false
+
+jobs:
+  build-and-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fix Dubious Ownership
+        run: git config --global --add safe.directory /opt/RoveComm_CPP
+
+      - name: Checkout Tag
+        run: |
+          cd /opt/RoveComm_CPP/
+          git fetch --tags --force
+          tag=$(git describe --tags `git rev-list --tags --max-count=1`)
+          echo $tag
+          git checkout $tag
+
+  create-release:
+    runs-on: ubuntu-latest
+    needs: build-and-upload
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fix Dubious Ownership
+        run: git config --global --add safe.directory /opt/RoveComm_CPP
+
+      - name: Determine Version
+        id: version
+        run: echo "version=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: false
+          prerelease: false
+          name: "${{ steps.version.outputs.version }}"
+          files: "/opt/RoveComm_CPP/build/**"
```

Comment:
The build directory won't exist in this context.

Reply:
I removed the sections that build the package since this is a library, but I figured it would be nice to have this action to easily create nice looking releases with changelogs and stuff.

Reply:
I removed the files line, don't know if the action will still work without it.

Reply:
I think it should work. I'll look at their docs real fast.

Reply:
Update: It should work without the files line.